courses/machine_learning/production_ml_systems/3_kubeflow_pipelines.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>We should skip this lab until this process is more streamlined.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Added that point to the timeline sheet. </p>